source/electrical power systems.md

```diff
@@ -0,0 +1,41 @@
+# Electrical Power Systems
+
+Electrical trains need to be powered.
+In almost all cases, the best way to do this is not by having trains carry their own energy supply, as diesel and coal trains do, but to be connected to the electrical grid to receive externally-generated power continuously.
+Such power systems can be broken down into overhead and "third rail" systems.
+
+## Overhead Catenary Wire
+
+Catenary wire is strung up over the tracks.
+
+Rolling stock which needs power is connected to the overhead wire with a *pantograph*.
+
+Modern overhead wire wire systems are typically at higher voltages AC, such as 25 kHz and 50 kHz.
+
+## Third Rail
+
+A "third rail" was originally just that, a third rail not used to support the train, but instead power it.
+(Modern third rail has diverged in material and shape from regular rail, but the name has stuck.)
+
+The third rail is much thicker than a regular wire, and as such can carry higher currents.
+At the same time, being right at ground level, and less well insulated, the third rail requires lower voltage.
+Lower voltage but higher current roughly cancels out in terms of power (their product), however.
+
+### Inferiority
+
+The physical power delivery mechanism itself are much more expensive for third wire.
+Third rails only last about [20 years](https://www.bart.gov/news/articles/2019/news20190731).
+Overhead wire itself and insulators, by contrast, typically lasts [over twice as long](https://www.sdtools.com/pdf/WCRR11_fatigue.pdf) as that, and the support poles often last even longer at over 100 years.
```

Comment:
Putting the whole point/subject/description in the hyperlinked texts makes it much easier for screen readers.  For example, you might ask a screen reader to list all of the links, and if it just says "20 years", you won't know what the link is, and URLs themselves are hard to read with a screen reader.

Reply:
https://jupyterbook.org/en/stable/content/citations.html Good point, and the above style is kinda kitschy anyways. We can also use the MyST syntax for citations for this.